app.rb

```diff
@@ -1,9 +1,31 @@
 require 'sinatra/base'
+require 'sinatra/reloader'
+require './lib/peep'
 
 class Chitter < Sinatra::Base
+  configure :development do
+  register Sinatra::Reloader
+end
+
   get '/test' do
     'Test page'
   end
 
+  get '/' do
+    @peeps = Peep.all
+    erb :peeps
+  end
+
+  post '/' do
+      message = params['peep']
+      connection = PG.connect(dbname: 'chitter_test')
```

Comment:
is it worth perhaps moving this DB connection to it's own object (class)? Rather than having it publicly exposed in the controller?